kpops/components/base_components/helm_app.py

```diff
@@ -68,12 +69,21 @@ class HelmApp(KubernetesApp):
         expects a path to local Helm chart.
     :param version: Helm chart version, defaults to None
     :param values: Helm app values
+    :param diff_config: Helm diff config
     """
 
+    # TODO: combine Helm config, e.g.
+    # helm:
+    #   repo: {}
+    #   diff: {}
```

Comment:
shall we do that now since we're making breaking changes anyways?

Reply:
Aren't they already combined because this is the Helm component? One could argue that values and version also belong to the helm config

Reply:
true, then I won't change it.